docker/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+if [ ! -f /config/qBittorrent/qBittorrent.conf ]; then
+    mkdir -p /config/qBittorrent/
+    cat << EOF > /config/qBittorrent/qBittorrent.conf
+[AutoRun]
+enabled=false
+program=
+[LegalNotice]
+Accepted=false
+[Preferences]
+Connection\UPnP=false
+Connection\PortRangeMin=6881
+Downloads\SavePath=/downloads/
+Downloads\ScanDirsV2=@Variant(\0\0\0\x1c\0\0\0\0)
```

Comment:
Why do you need such a legacy setting?

Reply:
In fact, you don't have to put anything there except what you want to set explicitly.

Reply:
nice, so I'll leave it empty except for the legal part

Reply:
>so I'll leave it empty except for the legal part

I suppose some of them are there for reasons. Better test it thoroughly.

Reply:
turns out qBittorrent auto-fill everything that's missing and needed, I'll just leave the legal option ant the defaults for the download directory, being a docker I wouldn't like users having to monkey with the settings as their first step just to get the downloaded files

everything else just gets added as needed (when the values are edited trough the web interface) or during qBittorrent first boot